### ISSUE TITLE
All stats page showing previous years stats in the month column.

### ISSUE THREAD

**Issue by wudanbal** [TheCraigHewitt/Seriously-Simple-Stats#52]
I've noticed this issue for quite some time, but wasn't sure if it was intended for not. But in the all stats pages view, the episode totals from the previous year appears in the view totals for each month. If this is not intended, I can roll a hot-fix for it.

**Comment by jonathanbossenger** [comment]
Hey @wudanbal 

Do you mean the monthly totals under the "All Episodes for the Last Three Months" section?

**Comment by wudanbal** [comment]
> Hey @wudanbal
> 
> Do you mean the monthly totals under the "All Episodes for the Last Three Months" section?

Yes, that's correct. I noticed it before, but wanted to point it out.

**Comment by thelovebug** [comment]
I confirm this bug.  I use SSP for (amongst other things) NaPodPoMo, which means most of my podcasting on this site is in the month of November.  I'm seeing last year's episodes in my "All Episodes for the Last Three Months" view, even though Stats for an Individual Episode shows nothing for this year.

**All stats**
![2019-11-12_10-00-40](https://user-images.githubusercontent.com/2915687/68662012-e0ba5400-0533-11ea-9634-587a3954b29c.png)

**Individual episode**
![2019-11-12_10-01-10](https://user-images.githubusercontent.com/2915687/68662028-e9128f00-0533-11ea-884c-3fab19cc1324.png)

**Comment by ivanguerrabimlevel** [comment]
I have the same problem. My podcast is one year old, and now I see that the January 2020 listeners of episode 1 are actually the January 2019 listeners plus the January 2020 listeners.

**Comment by jonathanbossenger** [comment]
Thanks everyone for the updates. Does this happen all the time, or just on a specific date. We had a similar bug which only happened on the first or last day of the month, so I just need to replicate this specific problem.

@wudanbal if you have the time to roll a PR/fix, that would also be helpful, as you have data where this is happening, and can therefore more easily test the fix.

**Comment by ivanguerrabimlevel** [comment]
It happens all the time, every time a new month of the second year begins.

Example: we just started April 2020, an episode I launched in February 2019 has:
February 2019 stats, ok
March 2019 stats, ok
April 2019 stats, ok
May 2019 stats, ok
...
January 2020 stats, ok
February 2020 stats, ERROR (February 2019+February 2020)
March  2020 stats, ERROR (March  2019+March 2020)
April  2020 stats, ERROR (April  2019+April 2020)

**Comment by jonathanbossenger** [comment]
@ivanguerrabimlevel that is very helpful, thank you. In that case I should be able to replicate the issue on my own podcast, as I have also released an episode in February.

Can you please confirm the search filters you apply to see the different sets of data (screenshots or a screencast would also be very helpful)

**Comment by ivanguerrabimlevel** [comment]
This problem does NOT occur in the graphics section, only in "All Episodes for the Last Three Months" section, so the filters do not matter.

Here is a snapshot of the current statistics: As you can see (in green), normal wiretapping is in the dozens, but when we enter the months where it was already published last year, wiretapping soars into the hundreds. (in red)
![Anotación 2020-04-14 094530](https://user-images.githubusercontent.com/55270125/79199862-dfb88600-7e35-11ea-9f73-876021a5898e.png)

**Comment by ivanguerrabimlevel** [comment]
PD: I haven't checked if this error affects the sum total, or the "top 10" section
